runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -331,9 +352,12 @@ ir::OperandIndex BaseLoader<LoaderDomain, SpecificLoader>::loadOperand(const Ten
 
   // Constant tensors are indicated by non-empty data.
   const auto *data = _model->buffers()->Get(tensor->buffer())->data();
+
+  int32_t pagesize = getpagesize();
```

Comment:
I expect the pagesize would not be changed while running, I think. If so(and I'm not wrong), the pagesize could be a static member of this class?

Reply:
@YongseopKim You're right. It will not change while running. I'll define `pagesize` as private member of BaseLoader. (I think it don't have to be static)